.github/workflows/protobuf-3.9.x.yml

```
@@ -0,0 +1,70 @@
+name: protobuf-3.9.x CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/protobuf/3.9.x/**'
+      - 'build.py'
+      - '.github/workflows/protobuf-3.9.x.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/protobuf/3.9.x/**'
+      - 'build.py'
+      - '.github/workflows/protobuf-3.9.x.yml'
+
+env:
+  CONAN_PASSWORD: ${{ secrets.BintrayApiKey }}
+
+jobs:
+  c-ares:
+    strategy:
+      matrix:
+        build: ['linux-gcc4.8', 'linux-gcc7']
+        protobuf_versions: ['3.9.1']
+        include:
+          - build: 'linux-gcc4.8'
+            os: 'ubuntu-latest'
+            compiler: 'gcc'
+            compiler_version: '4.8'
+            docker_image: 'conanio/gcc48'
+            build_types: 'Release,Debug'
+            archs: 'x86,x86_64'
+          - build: 'linux-gcc7'
+            os: 'ubuntu-latest'
+            compiler: 'gcc'
+            compiler_version: '7'
+            docker_image: 'conanio/gcc7'
+            build_types: 'Release,Debug'
+            archs: 'x86,x86_64'
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Configure git
+        run: git config --global core.autocrlf input
```

тут лучше использовать `false` - м договорились, что VCS не должна пытаться исправлять файлы
```suggestion
        run: git config --global core.autocrlf false
```